### ISSUE TITLE
Can I search the API using the command line?

### ISSUE THREAD

**Issue by Manuel-DominguezCBG** [broadinstitute/SpliceAI-lookup#40]
Question:

How can I query the API using exclusively the command line??

I would like to run the API remotely (via SSH) but I don't know to avoid using the browser. Is this possible (sorry first time working with APIs)

I have asked this here https://stackoverflow.com/questions/76343597/do-a-query-in-a-redis-api-with-the-terminal but I haven't  got an answer so far

**Comment by bw2** [comment]
Yes. The following steps work for most sites (not just this one):
1. Go to https://spliceailookup.broadinstitute.org/
2. In chrome, open Developer Tools:  <img width="616" alt="image" src="https://github.com/broadinstitute/SpliceAI-lookup/assets/6240170/ebc4d1ec-afd8-4c56-918e-d7ff04cae650">
3. Select the "Network" tab 
<img width="1156" alt="image" src="https://github.com/broadinstitute/SpliceAI-lookup/assets/6240170/bddb93aa-7ccc-45c8-8048-80050fce678c">
4. Use the website to search for a variant. The network tab will show all the requests the website made to the server: 
<img width="1035" alt="image" src="https://github.com/broadinstitute/SpliceAI-lookup/assets/6240170/ac9389c7-b21d-4616-93df-448e3e0bf13e">
5. Find the request that retrieves the info you're interested in (spliceai-lookup makes 2 separate requests to the api - one for spliceai scores, and one for pangolin): 
<img width="1661" alt="image" src="https://github.com/broadinstitute/SpliceAI-lookup/assets/6240170/b4c4e75c-264a-4b76-b6a9-15fafa94d5b1">
6. Right-click on the request row and go to the "Copy" menu. Select "Copy as cUrl:
<img width="1265" alt="image" src="https://github.com/broadinstitute/SpliceAI-lookup/assets/6240170/98e754c8-3f32-40c6-8fa2-38f0e4b17a31">
7. Paste into a linux or macos terminal, then run the command:
<img width="1423" alt="image" src="https://github.com/broadinstitute/SpliceAI-lookup/assets/6240170/525b282a-b6dd-4600-96a0-a52f2d9b87ef">
8. Most of the request headers are optional in this case. You can reduce it to just running 

```
curl 'https://spliceailookup-api.broadinstitute.org/spliceai/?hg=38&distance=500&mask=1&variant=8-140300616-T-G&raw=chr8-140300616-T-G'
```

and it should give you the same response. 

Alternatively, you can do this within a python script, using the requests library:
```
import requests
response = requests.get("https://spliceailookup-api.broadinstitute.org/spliceai/?hg=38&distance=500&mask=1&variant=8-140300616-T-G&raw=chr8-140300616-T-G", verify = False)
print(response.text)  # print without parsing
print(response.json())  # parse response as json
```
<img width="1586" alt="image" src="https://github.com/broadinstitute/SpliceAI-lookup/assets/6240170/f7a3c3e2-c02e-4287-8aef-8e47a24bee51">
